<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Q&A 4 How do I process raw sequencing data into a feature table using Mothur? | microbiome-viz.knit</title>
  <meta name="description" content="Microbiome Data Visualiztaion" />
  <meta name="generator" content="bookdown 0.42 and GitBook 2.6.7" />

  <meta property="og:title" content="Q&A 4 How do I process raw sequencing data into a feature table using Mothur? | microbiome-viz.knit" />
  <meta property="og:type" content="book" />
  
  
  <meta name="github-repo" content="https://github.com/cdi-eda/" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Q&A 4 How do I process raw sequencing data into a feature table using Mothur? | microbiome-viz.knit" />
  
  
  




  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  <link rel="shortcut icon" href="favicon.ico" type="image/x-icon" />
<link rel="prev" href="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-qiime2.html"/>
<link rel="next" href="how-do-i-explore-and-summarize-a-microbiome-otu-table.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { display: inline-block; text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { color: #008000; } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { color: #008000; font-weight: bold; } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

<style type="text/css">
  
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">CDI Data Science</a></li>

<li class="divider"></li>
<li><a href="index.html#section" id="toc-section"></a></li>
<li class="part"><span><b>I MICROBIOME DATA EDA</b></span></li>
<li class="chapter" data-level="1" data-path="what-are-the-essential-tools-for-microbiome-read-quality-control.html"><a href="what-are-the-essential-tools-for-microbiome-read-quality-control.html"><i class="fa fa-check"></i><b>1</b> What are the essential tools for microbiome read quality control?</a>
<ul>
<li class="chapter" data-level="1.1" data-path="what-are-the-essential-tools-for-microbiome-read-quality-control.html"><a href="what-are-the-essential-tools-for-microbiome-read-quality-control.html#explanation"><i class="fa fa-check"></i><b>1.1</b> Explanation</a></li>
<li class="chapter" data-level="1.2" data-path="what-are-the-essential-tools-for-microbiome-read-quality-control.html"><a href="what-are-the-essential-tools-for-microbiome-read-quality-control.html#shell-code"><i class="fa fa-check"></i><b>1.2</b> Shell Code</a></li>
<li class="chapter" data-level="1.3" data-path="what-are-the-essential-tools-for-microbiome-read-quality-control.html"><a href="what-are-the-essential-tools-for-microbiome-read-quality-control.html#r-note"><i class="fa fa-check"></i><b>1.3</b> R Note</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="how-do-i-obtain-example-microbiome-sequencing-data-for-analysis.html"><a href="how-do-i-obtain-example-microbiome-sequencing-data-for-analysis.html"><i class="fa fa-check"></i><b>2</b> How do I obtain example microbiome sequencing data for analysis?</a>
<ul>
<li class="chapter" data-level="2.1" data-path="how-do-i-obtain-example-microbiome-sequencing-data-for-analysis.html"><a href="how-do-i-obtain-example-microbiome-sequencing-data-for-analysis.html#explanation-1"><i class="fa fa-check"></i><b>2.1</b> Explanation</a></li>
<li class="chapter" data-level="2.2" data-path="how-do-i-obtain-example-microbiome-sequencing-data-for-analysis.html"><a href="how-do-i-obtain-example-microbiome-sequencing-data-for-analysis.html#shell-code-1"><i class="fa fa-check"></i><b>2.2</b> Shell Code</a></li>
<li class="chapter" data-level="2.3" data-path="how-do-i-obtain-example-microbiome-sequencing-data-for-analysis.html"><a href="how-do-i-obtain-example-microbiome-sequencing-data-for-analysis.html#python-note"><i class="fa fa-check"></i><b>2.3</b> Python Note</a></li>
<li class="chapter" data-level="2.4" data-path="how-do-i-obtain-example-microbiome-sequencing-data-for-analysis.html"><a href="how-do-i-obtain-example-microbiome-sequencing-data-for-analysis.html#r-note-1"><i class="fa fa-check"></i><b>2.4</b> R Note</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-qiime2.html"><a href="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-qiime2.html"><i class="fa fa-check"></i><b>3</b> How do I process raw sequencing data into a feature table using QIIME2?</a>
<ul>
<li class="chapter" data-level="3.1" data-path="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-qiime2.html"><a href="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-qiime2.html#explanation-2"><i class="fa fa-check"></i><b>3.1</b> Explanation</a></li>
<li class="chapter" data-level="3.2" data-path="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-qiime2.html"><a href="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-qiime2.html#shell-code-qiime2-cli"><i class="fa fa-check"></i><b>3.2</b> Shell Code (QIIME2 CLI)</a></li>
<li class="chapter" data-level="3.3" data-path="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-qiime2.html"><a href="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-qiime2.html#python-note-1"><i class="fa fa-check"></i><b>3.3</b> Python Note</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-mothur.html"><a href="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-mothur.html"><i class="fa fa-check"></i><b>4</b> How do I process raw sequencing data into a feature table using Mothur?</a>
<ul>
<li class="chapter" data-level="4.1" data-path="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-mothur.html"><a href="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-mothur.html#explanation-3"><i class="fa fa-check"></i><b>4.1</b> Explanation</a></li>
<li class="chapter" data-level="4.2" data-path="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-mothur.html"><a href="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-mothur.html#shell-code-2"><i class="fa fa-check"></i><b>4.2</b> Shell Code</a></li>
<li class="chapter" data-level="4.3" data-path="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-mothur.html"><a href="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-mothur.html#r-note-2"><i class="fa fa-check"></i><b>4.3</b> R Note</a></li>
<li class="chapter" data-level="4.4" data-path="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-mothur.html"><a href="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-mothur.html#python-note-2"><i class="fa fa-check"></i><b>4.4</b> Python Note</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="how-do-i-explore-and-summarize-a-microbiome-otu-table.html"><a href="how-do-i-explore-and-summarize-a-microbiome-otu-table.html"><i class="fa fa-check"></i><b>5</b> How do I explore and summarize a microbiome OTU table?</a>
<ul>
<li class="chapter" data-level="5.1" data-path="how-do-i-explore-and-summarize-a-microbiome-otu-table.html"><a href="how-do-i-explore-and-summarize-a-microbiome-otu-table.html#explanation-4"><i class="fa fa-check"></i><b>5.1</b> Explanation</a></li>
<li class="chapter" data-level="5.2" data-path="how-do-i-explore-and-summarize-a-microbiome-otu-table.html"><a href="how-do-i-explore-and-summarize-a-microbiome-otu-table.html#python-code"><i class="fa fa-check"></i><b>5.2</b> Python Code</a></li>
<li class="chapter" data-level="5.3" data-path="how-do-i-explore-and-summarize-a-microbiome-otu-table.html"><a href="how-do-i-explore-and-summarize-a-microbiome-otu-table.html#r-code"><i class="fa fa-check"></i><b>5.3</b> R Code</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="how-do-i-filter-out-low-abundance-or-low-prevalence-otus.html"><a href="how-do-i-filter-out-low-abundance-or-low-prevalence-otus.html"><i class="fa fa-check"></i><b>6</b> How do I filter out low-abundance or low-prevalence OTUs?</a>
<ul>
<li class="chapter" data-level="6.1" data-path="how-do-i-filter-out-low-abundance-or-low-prevalence-otus.html"><a href="how-do-i-filter-out-low-abundance-or-low-prevalence-otus.html#explanation-5"><i class="fa fa-check"></i><b>6.1</b> Explanation</a></li>
<li class="chapter" data-level="6.2" data-path="how-do-i-filter-out-low-abundance-or-low-prevalence-otus.html"><a href="how-do-i-filter-out-low-abundance-or-low-prevalence-otus.html#python-code-1"><i class="fa fa-check"></i><b>6.2</b> Python Code</a></li>
<li class="chapter" data-level="6.3" data-path="how-do-i-filter-out-low-abundance-or-low-prevalence-otus.html"><a href="how-do-i-filter-out-low-abundance-or-low-prevalence-otus.html#r-code-1"><i class="fa fa-check"></i><b>6.3</b> R Code</a></li>
</ul></li>
<li class="part"><span><b>II MICROBIOME DATA VISUALIZATION</b></span></li>
<li class="chapter" data-level="7" data-path="how-do-i-visualize-total-otu-abundance-per-sample.html"><a href="how-do-i-visualize-total-otu-abundance-per-sample.html"><i class="fa fa-check"></i><b>7</b> How do I visualize total OTU abundance per sample?</a>
<ul>
<li class="chapter" data-level="7.1" data-path="how-do-i-visualize-total-otu-abundance-per-sample.html"><a href="how-do-i-visualize-total-otu-abundance-per-sample.html#explanation-6"><i class="fa fa-check"></i><b>7.1</b> Explanation</a></li>
<li class="chapter" data-level="7.2" data-path="how-do-i-visualize-total-otu-abundance-per-sample.html"><a href="how-do-i-visualize-total-otu-abundance-per-sample.html#python-code-2"><i class="fa fa-check"></i><b>7.2</b> Python Code</a></li>
<li class="chapter" data-level="7.3" data-path="how-do-i-visualize-total-otu-abundance-per-sample.html"><a href="how-do-i-visualize-total-otu-abundance-per-sample.html#r-code-2"><i class="fa fa-check"></i><b>7.3</b> R Code</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="how-do-i-create-a-stacked-bar-plot-of-top-genera-across-samples.html"><a href="how-do-i-create-a-stacked-bar-plot-of-top-genera-across-samples.html"><i class="fa fa-check"></i><b>8</b> How do I create a stacked bar plot of top genera across samples?</a>
<ul>
<li class="chapter" data-level="8.1" data-path="how-do-i-create-a-stacked-bar-plot-of-top-genera-across-samples.html"><a href="how-do-i-create-a-stacked-bar-plot-of-top-genera-across-samples.html#explanation-7"><i class="fa fa-check"></i><b>8.1</b> Explanation</a></li>
<li class="chapter" data-level="8.2" data-path="how-do-i-create-a-stacked-bar-plot-of-top-genera-across-samples.html"><a href="how-do-i-create-a-stacked-bar-plot-of-top-genera-across-samples.html#python-code-3"><i class="fa fa-check"></i><b>8.2</b> Python Code</a></li>
<li class="chapter" data-level="8.3" data-path="how-do-i-create-a-stacked-bar-plot-of-top-genera-across-samples.html"><a href="how-do-i-create-a-stacked-bar-plot-of-top-genera-across-samples.html#r-code-3"><i class="fa fa-check"></i><b>8.3</b> R Code</a></li>
</ul></li>
<li class="chapter" data-level="9" data-path="how-do-i-visualize-alpha-diversity-richness-across-groups.html"><a href="how-do-i-visualize-alpha-diversity-richness-across-groups.html"><i class="fa fa-check"></i><b>9</b> How do I visualize alpha diversity (richness) across groups?</a>
<ul>
<li class="chapter" data-level="9.1" data-path="how-do-i-visualize-alpha-diversity-richness-across-groups.html"><a href="how-do-i-visualize-alpha-diversity-richness-across-groups.html#explanation-8"><i class="fa fa-check"></i><b>9.1</b> Explanation</a></li>
<li class="chapter" data-level="9.2" data-path="how-do-i-visualize-alpha-diversity-richness-across-groups.html"><a href="how-do-i-visualize-alpha-diversity-richness-across-groups.html#python-code-4"><i class="fa fa-check"></i><b>9.2</b> Python Code</a></li>
<li class="chapter" data-level="9.3" data-path="how-do-i-visualize-alpha-diversity-richness-across-groups.html"><a href="how-do-i-visualize-alpha-diversity-richness-across-groups.html#r-code-4"><i class="fa fa-check"></i><b>9.3</b> R Code</a></li>
</ul></li>
<li class="chapter" data-level="10" data-path="how-do-i-perform-ordination-e.g.-pca-to-visualize-sample-clustering.html"><a href="how-do-i-perform-ordination-e.g.-pca-to-visualize-sample-clustering.html"><i class="fa fa-check"></i><b>10</b> How do I perform ordination (e.g., PCA) to visualize sample clustering?</a>
<ul>
<li class="chapter" data-level="10.1" data-path="how-do-i-perform-ordination-e.g.-pca-to-visualize-sample-clustering.html"><a href="how-do-i-perform-ordination-e.g.-pca-to-visualize-sample-clustering.html#explanation-9"><i class="fa fa-check"></i><b>10.1</b> Explanation</a></li>
<li class="chapter" data-level="10.2" data-path="how-do-i-perform-ordination-e.g.-pca-to-visualize-sample-clustering.html"><a href="how-do-i-perform-ordination-e.g.-pca-to-visualize-sample-clustering.html#python-code-5"><i class="fa fa-check"></i><b>10.2</b> Python Code</a></li>
<li class="chapter" data-level="10.3" data-path="how-do-i-perform-ordination-e.g.-pca-to-visualize-sample-clustering.html"><a href="how-do-i-perform-ordination-e.g.-pca-to-visualize-sample-clustering.html#r-code-5"><i class="fa fa-check"></i><b>10.3</b> R Code</a></li>
</ul></li>
<li class="chapter" data-level="11" data-path="how-do-i-visualize-otu-or-genus-abundance-using-a-heatmap.html"><a href="how-do-i-visualize-otu-or-genus-abundance-using-a-heatmap.html"><i class="fa fa-check"></i><b>11</b> How do I visualize OTU or Genus abundance using a heatmap?</a>
<ul>
<li class="chapter" data-level="11.1" data-path="how-do-i-visualize-otu-or-genus-abundance-using-a-heatmap.html"><a href="how-do-i-visualize-otu-or-genus-abundance-using-a-heatmap.html#explanation-10"><i class="fa fa-check"></i><b>11.1</b> Explanation</a></li>
<li class="chapter" data-level="11.2" data-path="how-do-i-visualize-otu-or-genus-abundance-using-a-heatmap.html"><a href="how-do-i-visualize-otu-or-genus-abundance-using-a-heatmap.html#python-code-6"><i class="fa fa-check"></i><b>11.2</b> Python Code</a></li>
<li class="chapter" data-level="11.3" data-path="how-do-i-visualize-otu-or-genus-abundance-using-a-heatmap.html"><a href="how-do-i-visualize-otu-or-genus-abundance-using-a-heatmap.html#r-code-6"><i class="fa fa-check"></i><b>11.3</b> R Code</a></li>
</ul></li>
<li class="divider"></li>
<li><a href="https://complexdatainsights.com/product">Explore More Guides</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./"><div class="line-block">Microbiome Data Visualiztaion</div></a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-mothur" class="section level1 hasAnchor" number="4">
<h1><span class="header-section-number">Q&A 4</span> How do I process raw sequencing data into a feature table using Mothur?<a href="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-mothur.html#how-do-i-process-raw-sequencing-data-into-a-feature-table-using-mothur" class="anchor-section" aria-label="Anchor link to header"></a></h1>
<div id="explanation-3" class="section level2 hasAnchor" number="4.1">
<h2><span class="header-section-number">4.1</span> Explanation<a href="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-mothur.html#explanation-3" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>Mothur is an open-source software package for analyzing 16S rRNA gene sequences. It supports raw data preprocessing, OTU clustering, and taxonomic assignment. Its workflow is particularly popular for microbial community studies and works well with single- or paired-end FASTQ data.</p>
<p>Mothur pipelines are usually run using command files or interactively within the Mothur console. Key steps include:
- Merging paired-end reads
- Quality filtering
- Aligning and clustering reads into OTUs
- Generating <code>.shared</code> (feature table) and <code>.taxonomy</code> files</p>
</div>
<div id="shell-code-2" class="section level2 hasAnchor" number="4.2">
<h2><span class="header-section-number">4.2</span> Shell Code<a href="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-mothur.html#shell-code-2" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<div class="sourceCode" id="cb8"><pre class="sourceCode bash"><code class="sourceCode bash"><span id="cb8-1"><a href="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-mothur.html#cb8-1" tabindex="-1"></a><span class="co"># Launch Mothur</span></span>
<span id="cb8-2"><a href="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-mothur.html#cb8-2" tabindex="-1"></a><span class="ex">mothur</span></span>
<span id="cb8-3"><a href="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-mothur.html#cb8-3" tabindex="-1"></a></span>
<span id="cb8-4"><a href="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-mothur.html#cb8-4" tabindex="-1"></a><span class="co"># Inside Mothur console (example workflow)</span></span>
<span id="cb8-5"><a href="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-mothur.html#cb8-5" tabindex="-1"></a><span class="ex">mothur</span> <span class="op">&gt;</span> make.file<span class="er">(</span><span class="va">inputdir</span><span class="op">=</span>./raw_data, <span class="va">type</span><span class="op">=</span>fastq, <span class="va">prefix</span><span class="op">=</span>stability<span class="kw">)</span></span>
<span id="cb8-6"><a href="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-mothur.html#cb8-6" tabindex="-1"></a><span class="ex">mothur</span> <span class="op">&gt;</span> make.contigs<span class="er">(</span><span class="va">file</span><span class="op">=</span>stability.files, <span class="va">processors</span><span class="op">=</span>8<span class="kw">)</span></span>
<span id="cb8-7"><a href="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-mothur.html#cb8-7" tabindex="-1"></a><span class="ex">mothur</span> <span class="op">&gt;</span> screen.seqs<span class="er">(</span><span class="va">fasta</span><span class="op">=</span>stability.trim.contigs.fasta, <span class="va">group</span><span class="op">=</span>stability.contigs.groups, <span class="va">maxambig</span><span class="op">=</span>0, <span class="va">maxlength</span><span class="op">=</span>275<span class="kw">)</span></span>
<span id="cb8-8"><a href="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-mothur.html#cb8-8" tabindex="-1"></a><span class="ex">mothur</span> <span class="op">&gt;</span> unique.seqs<span class="er">(</span><span class="va">fasta</span><span class="op">=</span>stability.trim.contigs.good.fasta<span class="kw">)</span></span>
<span id="cb8-9"><a href="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-mothur.html#cb8-9" tabindex="-1"></a><span class="ex">mothur</span> <span class="op">&gt;</span> count.seqs<span class="er">(</span><span class="va">name</span><span class="op">=</span>stability.trim.contigs.good.names, <span class="va">group</span><span class="op">=</span>stability.contigs.good.groups<span class="kw">)</span></span>
<span id="cb8-10"><a href="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-mothur.html#cb8-10" tabindex="-1"></a><span class="ex">mothur</span> <span class="op">&gt;</span> align.seqs<span class="er">(</span><span class="va">fasta</span><span class="op">=</span>stability.trim.contigs.good.unique.fasta, <span class="va">reference</span><span class="op">=</span>silva.seed_v138.align<span class="kw">)</span></span>
<span id="cb8-11"><a href="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-mothur.html#cb8-11" tabindex="-1"></a><span class="ex">mothur</span> <span class="op">&gt;</span> screen.seqs<span class="er">(</span><span class="ex">...</span><span class="kw">)</span></span>
<span id="cb8-12"><a href="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-mothur.html#cb8-12" tabindex="-1"></a><span class="ex">mothur</span> <span class="op">&gt;</span> pre.cluster<span class="er">(</span><span class="ex">...</span><span class="kw">)</span></span>
<span id="cb8-13"><a href="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-mothur.html#cb8-13" tabindex="-1"></a><span class="ex">mothur</span> <span class="op">&gt;</span> chimera.vsearch<span class="er">(</span><span class="ex">...</span><span class="kw">)</span></span>
<span id="cb8-14"><a href="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-mothur.html#cb8-14" tabindex="-1"></a><span class="ex">mothur</span> <span class="op">&gt;</span> remove.seqs<span class="er">(</span><span class="ex">...</span><span class="kw">)</span></span>
<span id="cb8-15"><a href="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-mothur.html#cb8-15" tabindex="-1"></a><span class="ex">mothur</span> <span class="op">&gt;</span> cluster.split<span class="er">(</span><span class="ex">...</span><span class="kw">)</span></span>
<span id="cb8-16"><a href="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-mothur.html#cb8-16" tabindex="-1"></a><span class="ex">mothur</span> <span class="op">&gt;</span> make.shared<span class="er">(</span><span class="ex">...</span><span class="kw">)</span></span>
<span id="cb8-17"><a href="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-mothur.html#cb8-17" tabindex="-1"></a><span class="ex">mothur</span> <span class="op">&gt;</span> classify.otu<span class="er">(</span><span class="ex">...</span><span class="kw">)</span></span></code></pre></div>
</div>
<div id="r-note-2" class="section level2 hasAnchor" number="4.3">
<h2><span class="header-section-number">4.3</span> R Note<a href="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-mothur.html#r-note-2" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<div class="sourceCode" id="cb9"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb9-1"><a href="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-mothur.html#cb9-1" tabindex="-1"></a><span class="co"># Mothur outputs a `.shared` file (feature/OTU table) and `.taxonomy` file.</span></span>
<span id="cb9-2"><a href="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-mothur.html#cb9-2" tabindex="-1"></a><span class="co"># These can be imported into R using packages like phyloseq or tidyverse for analysis.</span></span></code></pre></div>
</div>
<div id="python-note-2" class="section level2 hasAnchor" number="4.4">
<h2><span class="header-section-number">4.4</span> Python Note<a href="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-mothur.html#python-note-2" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<div class="sourceCode" id="cb10"><pre class="sourceCode python"><code class="sourceCode python"><span id="cb10-1"><a href="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-mothur.html#cb10-1" tabindex="-1"></a><span class="co"># Mothur is not Python-based, but output files can be parsed with pandas or biom-format readers.</span></span></code></pre></div>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-qiime2.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="how-do-i-explore-and-summarize-a-microbiome-otu-table.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
  "sharing": {
    "github": false,
    "facebook": true,
    "twitter": true,
    "linkedin": false,
    "weibo": false,
    "instapaper": false,
    "vk": false,
    "whatsapp": false,
    "all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
  },
  "fontsettings": {
    "theme": "white",
    "family": "sans",
    "size": 2
  },
  "edit": {
    "link": "https://github.com/datainsights/cdi-framework/edit/main/02-1-microb-eda.Rmd",
    "text": "Edit"
  },
  "history": {
    "link": null,
    "text": null
  },
  "view": {
    "link": null,
    "text": null
  },
  "download": ["microbiome-viz.pdf", "microbiome-viz.epub"],
  "search": {
    "engine": "fuse",
    "options": null
  },
  "toc": {
    "collapse": "subsection"
  }
});
});
</script>

</body>

</html>
