<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Q&A 3 How do I process raw sequencing data into a feature table using QIIME2? | microbiome-eda.knit</title>
  <meta name="description" content="Microbiome Exploratory Data Analysis" />
  <meta name="generator" content="bookdown 0.42 and GitBook 2.6.7" />

  <meta property="og:title" content="Q&A 3 How do I process raw sequencing data into a feature table using QIIME2? | microbiome-eda.knit" />
  <meta property="og:type" content="book" />
  
  
  <meta name="github-repo" content="https://github.com/cdi-microbiome/" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Q&A 3 How do I process raw sequencing data into a feature table using QIIME2? | microbiome-eda.knit" />
  
  
  




  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  <link rel="shortcut icon" href="favicon.ico" type="image/x-icon" />
<link rel="prev" href="how-do-i-obtain-example-microbiome-sequencing-data-for-analysis.html"/>
<link rel="next" href="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-mothur.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { display: inline-block; text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { color: #008000; } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { color: #008000; font-weight: bold; } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

<style type="text/css">
  
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="https://microbiome.complexdatainsights.com/">Microbiome Analysis</a></li>

<li class="divider"></li>
<li><a href="index.html#section" id="toc-section"></a></li>
<li class="part"><span><b>I MICROBIOME DATA EDA</b></span></li>
<li class="chapter" data-level="1" data-path="what-are-the-essential-tools-for-microbiome-read-quality-control.html"><a href="what-are-the-essential-tools-for-microbiome-read-quality-control.html"><i class="fa fa-check"></i><b>1</b> What are the essential tools for microbiome read quality control?</a>
<ul>
<li class="chapter" data-level="1.1" data-path="what-are-the-essential-tools-for-microbiome-read-quality-control.html"><a href="what-are-the-essential-tools-for-microbiome-read-quality-control.html#explanation"><i class="fa fa-check"></i><b>1.1</b> Explanation</a></li>
<li class="chapter" data-level="1.2" data-path="what-are-the-essential-tools-for-microbiome-read-quality-control.html"><a href="what-are-the-essential-tools-for-microbiome-read-quality-control.html#shell-code"><i class="fa fa-check"></i><b>1.2</b> Shell Code</a></li>
<li class="chapter" data-level="1.3" data-path="what-are-the-essential-tools-for-microbiome-read-quality-control.html"><a href="what-are-the-essential-tools-for-microbiome-read-quality-control.html#r-note"><i class="fa fa-check"></i><b>1.3</b> R Note</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="how-do-i-obtain-example-microbiome-sequencing-data-for-analysis.html"><a href="how-do-i-obtain-example-microbiome-sequencing-data-for-analysis.html"><i class="fa fa-check"></i><b>2</b> How do I obtain example microbiome sequencing data for analysis?</a>
<ul>
<li class="chapter" data-level="2.1" data-path="how-do-i-obtain-example-microbiome-sequencing-data-for-analysis.html"><a href="how-do-i-obtain-example-microbiome-sequencing-data-for-analysis.html#explanation-1"><i class="fa fa-check"></i><b>2.1</b> Explanation</a></li>
<li class="chapter" data-level="2.2" data-path="how-do-i-obtain-example-microbiome-sequencing-data-for-analysis.html"><a href="how-do-i-obtain-example-microbiome-sequencing-data-for-analysis.html#shell-code-1"><i class="fa fa-check"></i><b>2.2</b> Shell Code</a></li>
<li class="chapter" data-level="2.3" data-path="how-do-i-obtain-example-microbiome-sequencing-data-for-analysis.html"><a href="how-do-i-obtain-example-microbiome-sequencing-data-for-analysis.html#python-note"><i class="fa fa-check"></i><b>2.3</b> Python Note</a></li>
<li class="chapter" data-level="2.4" data-path="how-do-i-obtain-example-microbiome-sequencing-data-for-analysis.html"><a href="how-do-i-obtain-example-microbiome-sequencing-data-for-analysis.html#r-note-1"><i class="fa fa-check"></i><b>2.4</b> R Note</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-qiime2.html"><a href="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-qiime2.html"><i class="fa fa-check"></i><b>3</b> How do I process raw sequencing data into a feature table using QIIME2?</a>
<ul>
<li class="chapter" data-level="3.1" data-path="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-qiime2.html"><a href="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-qiime2.html#explanation-2"><i class="fa fa-check"></i><b>3.1</b> Explanation</a></li>
<li class="chapter" data-level="3.2" data-path="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-qiime2.html"><a href="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-qiime2.html#shell-code-qiime2-cli"><i class="fa fa-check"></i><b>3.2</b> Shell Code (QIIME2 CLI)</a></li>
<li class="chapter" data-level="3.3" data-path="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-qiime2.html"><a href="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-qiime2.html#python-note-1"><i class="fa fa-check"></i><b>3.3</b> Python Note</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-mothur.html"><a href="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-mothur.html"><i class="fa fa-check"></i><b>4</b> How do I process raw sequencing data into a feature table using Mothur?</a>
<ul>
<li class="chapter" data-level="4.1" data-path="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-mothur.html"><a href="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-mothur.html#explanation-3"><i class="fa fa-check"></i><b>4.1</b> Explanation</a></li>
<li class="chapter" data-level="4.2" data-path="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-mothur.html"><a href="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-mothur.html#shell-code-2"><i class="fa fa-check"></i><b>4.2</b> Shell Code</a></li>
<li class="chapter" data-level="4.3" data-path="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-mothur.html"><a href="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-mothur.html#r-note-2"><i class="fa fa-check"></i><b>4.3</b> R Note</a></li>
<li class="chapter" data-level="4.4" data-path="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-mothur.html"><a href="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-mothur.html#python-note-2"><i class="fa fa-check"></i><b>4.4</b> Python Note</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="how-do-i-explore-and-summarize-a-microbiome-otu-table.html"><a href="how-do-i-explore-and-summarize-a-microbiome-otu-table.html"><i class="fa fa-check"></i><b>5</b> How do I explore and summarize a microbiome OTU table?</a>
<ul>
<li class="chapter" data-level="5.1" data-path="how-do-i-explore-and-summarize-a-microbiome-otu-table.html"><a href="how-do-i-explore-and-summarize-a-microbiome-otu-table.html#explanation-4"><i class="fa fa-check"></i><b>5.1</b> Explanation</a></li>
<li class="chapter" data-level="5.2" data-path="how-do-i-explore-and-summarize-a-microbiome-otu-table.html"><a href="how-do-i-explore-and-summarize-a-microbiome-otu-table.html#python-code"><i class="fa fa-check"></i><b>5.2</b> Python Code</a></li>
<li class="chapter" data-level="5.3" data-path="how-do-i-explore-and-summarize-a-microbiome-otu-table.html"><a href="how-do-i-explore-and-summarize-a-microbiome-otu-table.html#r-code"><i class="fa fa-check"></i><b>5.3</b> R Code</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="how-do-i-filter-out-low-abundance-or-low-prevalence-otus.html"><a href="how-do-i-filter-out-low-abundance-or-low-prevalence-otus.html"><i class="fa fa-check"></i><b>6</b> How do I filter out low-abundance or low-prevalence OTUs?</a>
<ul>
<li class="chapter" data-level="6.1" data-path="how-do-i-filter-out-low-abundance-or-low-prevalence-otus.html"><a href="how-do-i-filter-out-low-abundance-or-low-prevalence-otus.html#explanation-5"><i class="fa fa-check"></i><b>6.1</b> Explanation</a></li>
<li class="chapter" data-level="6.2" data-path="how-do-i-filter-out-low-abundance-or-low-prevalence-otus.html"><a href="how-do-i-filter-out-low-abundance-or-low-prevalence-otus.html#python-code-1"><i class="fa fa-check"></i><b>6.2</b> Python Code</a></li>
<li class="chapter" data-level="6.3" data-path="how-do-i-filter-out-low-abundance-or-low-prevalence-otus.html"><a href="how-do-i-filter-out-low-abundance-or-low-prevalence-otus.html#r-code-1"><i class="fa fa-check"></i><b>6.3</b> R Code</a></li>
</ul></li>
<li class="divider"></li>
<li><a href="https://complexdatainsights.com/product">Explore More Guides</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./"><div class="line-block">Microbiome Exploratory Data Analysis</div></a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-qiime2" class="section level1 hasAnchor" number="3">
<h1><span class="header-section-number">Q&A 3</span> How do I process raw sequencing data into a feature table using QIIME2?<a href="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-qiime2.html#how-do-i-process-raw-sequencing-data-into-a-feature-table-using-qiime2" class="anchor-section" aria-label="Anchor link to header"></a></h1>
<div id="explanation-2" class="section level2 hasAnchor" number="3.1">
<h2><span class="header-section-number">3.1</span> Explanation<a href="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-qiime2.html#explanation-2" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>After obtaining your raw FASTQ data, the first analytical step is transforming it into a structured format for analysis — the <strong>feature table</strong>. This is a matrix of counts (samples × ASVs or OTUs).</p>
<p>QIIME2 is a powerful, Python-based platform for this entire workflow. It uses <code>.qza</code> files (QIIME artifacts) to structure data at each step and generates a <code>.qzv</code> summary for inspection.</p>
<p>This pipeline includes:
- Importing FASTQ data
- Demultiplexing reads
- Denoising to generate ASVs using <code>DADA2</code> or <code>Deblur</code>
- Creating a feature table</p>
</div>
<div id="shell-code-qiime2-cli" class="section level2 hasAnchor" number="3.2">
<h2><span class="header-section-number">3.2</span> Shell Code (QIIME2 CLI)<a href="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-qiime2.html#shell-code-qiime2-cli" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<div class="sourceCode" id="cb6"><pre class="sourceCode bash"><code class="sourceCode bash"><span id="cb6-1"><a href="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-qiime2.html#cb6-1" tabindex="-1"></a><span class="co"># 1. Import paired-end reads</span></span>
<span id="cb6-2"><a href="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-qiime2.html#cb6-2" tabindex="-1"></a><span class="ex">qiime</span> tools import <span class="dt">\</span></span>
<span id="cb6-3"><a href="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-qiime2.html#cb6-3" tabindex="-1"></a>  <span class="at">--type</span> EMPPairedEndSequences <span class="dt">\</span></span>
<span id="cb6-4"><a href="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-qiime2.html#cb6-4" tabindex="-1"></a>  <span class="at">--input-path</span> emp-paired-end-sequences <span class="dt">\</span></span>
<span id="cb6-5"><a href="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-qiime2.html#cb6-5" tabindex="-1"></a>  <span class="at">--output-path</span> emp-paired-end-sequences.qza</span>
<span id="cb6-6"><a href="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-qiime2.html#cb6-6" tabindex="-1"></a></span>
<span id="cb6-7"><a href="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-qiime2.html#cb6-7" tabindex="-1"></a><span class="co"># 2. Demultiplex (barcode + sample mapping required)</span></span>
<span id="cb6-8"><a href="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-qiime2.html#cb6-8" tabindex="-1"></a><span class="ex">qiime</span> demux emp-paired <span class="dt">\</span></span>
<span id="cb6-9"><a href="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-qiime2.html#cb6-9" tabindex="-1"></a>  <span class="at">--i-seqs</span> emp-paired-end-sequences.qza <span class="dt">\</span></span>
<span id="cb6-10"><a href="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-qiime2.html#cb6-10" tabindex="-1"></a>  <span class="at">--m-barcodes-file</span> sample-metadata.tsv <span class="dt">\</span></span>
<span id="cb6-11"><a href="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-qiime2.html#cb6-11" tabindex="-1"></a>  <span class="at">--m-barcodes-column</span> BarcodeSequence <span class="dt">\</span></span>
<span id="cb6-12"><a href="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-qiime2.html#cb6-12" tabindex="-1"></a>  <span class="at">--o-per-sample-sequences</span> demux.qza <span class="dt">\</span></span>
<span id="cb6-13"><a href="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-qiime2.html#cb6-13" tabindex="-1"></a>  <span class="at">--o-error-correction-details</span> demux-details.qza</span>
<span id="cb6-14"><a href="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-qiime2.html#cb6-14" tabindex="-1"></a></span>
<span id="cb6-15"><a href="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-qiime2.html#cb6-15" tabindex="-1"></a><span class="co"># 3. Visualize quality</span></span>
<span id="cb6-16"><a href="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-qiime2.html#cb6-16" tabindex="-1"></a><span class="ex">qiime</span> demux summarize <span class="dt">\</span></span>
<span id="cb6-17"><a href="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-qiime2.html#cb6-17" tabindex="-1"></a>  <span class="at">--i-data</span> demux.qza <span class="dt">\</span></span>
<span id="cb6-18"><a href="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-qiime2.html#cb6-18" tabindex="-1"></a>  <span class="at">--o-visualization</span> demux.qzv</span>
<span id="cb6-19"><a href="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-qiime2.html#cb6-19" tabindex="-1"></a></span>
<span id="cb6-20"><a href="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-qiime2.html#cb6-20" tabindex="-1"></a><span class="co"># 4. Denoise with DADA2</span></span>
<span id="cb6-21"><a href="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-qiime2.html#cb6-21" tabindex="-1"></a><span class="ex">qiime</span> dada2 denoise-paired <span class="dt">\</span></span>
<span id="cb6-22"><a href="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-qiime2.html#cb6-22" tabindex="-1"></a>  <span class="at">--i-demultiplexed-seqs</span> demux.qza <span class="dt">\</span></span>
<span id="cb6-23"><a href="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-qiime2.html#cb6-23" tabindex="-1"></a>  <span class="at">--p-trim-left-f</span> 0 <span class="dt">\</span></span>
<span id="cb6-24"><a href="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-qiime2.html#cb6-24" tabindex="-1"></a>  <span class="at">--p-trim-left-r</span> 0 <span class="dt">\</span></span>
<span id="cb6-25"><a href="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-qiime2.html#cb6-25" tabindex="-1"></a>  <span class="at">--p-trunc-len-f</span> 240 <span class="dt">\</span></span>
<span id="cb6-26"><a href="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-qiime2.html#cb6-26" tabindex="-1"></a>  <span class="at">--p-trunc-len-r</span> 200 <span class="dt">\</span></span>
<span id="cb6-27"><a href="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-qiime2.html#cb6-27" tabindex="-1"></a>  <span class="at">--o-table</span> table.qza <span class="dt">\</span></span>
<span id="cb6-28"><a href="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-qiime2.html#cb6-28" tabindex="-1"></a>  <span class="at">--o-representative-sequences</span> rep-seqs.qza <span class="dt">\</span></span>
<span id="cb6-29"><a href="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-qiime2.html#cb6-29" tabindex="-1"></a>  <span class="at">--o-denoising-stats</span> denoising-stats.qza</span></code></pre></div>
</div>
<div id="python-note-1" class="section level2 hasAnchor" number="3.3">
<h2><span class="header-section-number">3.3</span> Python Note<a href="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-qiime2.html#python-note-1" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<div class="sourceCode" id="cb7"><pre class="sourceCode python"><code class="sourceCode python"><span id="cb7-1"><a href="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-qiime2.html#cb7-1" tabindex="-1"></a><span class="co"># Although QIIME2 is Python-based, its workflow is run via CLI.</span></span>
<span id="cb7-2"><a href="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-qiime2.html#cb7-2" tabindex="-1"></a><span class="co"># Feature table (table.qza) can be exported and summarized later using Python or R.</span></span></code></pre></div>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="how-do-i-obtain-example-microbiome-sequencing-data-for-analysis.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="how-do-i-process-raw-sequencing-data-into-a-feature-table-using-mothur.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
  "sharing": {
    "github": false,
    "facebook": true,
    "twitter": true,
    "linkedin": false,
    "weibo": false,
    "instapaper": false,
    "vk": false,
    "whatsapp": false,
    "all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
  },
  "fontsettings": {
    "theme": "white",
    "family": "sans",
    "size": 2
  },
  "edit": {
    "link": "https://github.com/datainsights/cdi-microbiome/edit/main/02-1-microb-eda.Rmd",
    "text": "Edit"
  },
  "history": {
    "link": null,
    "text": null
  },
  "view": {
    "link": null,
    "text": null
  },
  "download": ["microbiome-eda.pdf", "microbiome-eda.epub"],
  "search": {
    "engine": "fuse",
    "options": null
  },
  "toc": {
    "collapse": "subsection"
  }
});
});
</script>

</body>

</html>
